tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -45,6 +45,12 @@ sub run_test {
         assert_script_run("test $MEM -ge 20", fail_message => "The SUT needs at least 20G FREE MEM for virtual network test");
     }
 
+    #After deployed guest systems, ensure default pool have at least
+    #60Gib available disk space on vm host for virtual network test
+    my $POOL = $self->get_pool_available_space();
+    record_info('Detect Available POOL', $POOL . 'Gib');
+    assert_script_run("test $POOL -ge 60", fail_message => "The SUT needs at least 60Gib available space of default pool for virtual network test");
```

Comment:
Since `$pool` is available as an internal variable here, why not move the step away from the openQA virtual machine and do something like

    die("The SUT needs at least 60Gib available space of default pool for virtual network test") if ($pool < 60);

instead of the rather slow `assert_script_run`?

Reply:
@grisu48 thanks so much for your comment. 
we had discuss about using assert_script_run() and die() already.

At last, we had agreed on  using our OSD API - assert_script_run() here, instead of die().  

Thus, there was used assert_script_run().

Reply:
Ok I see. I don't fully get the advantage of using `assert_script_run` here, but consider this a minor issue so feel free to proceed.